CoronaContact/Extensions/ENExposureDetectionSummary+totalRiskScore.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ENExposureDetectionSummary+totalRiskScore.swift
+//  CoronaContact
+//
+
+import ExposureNotification
+import Foundation
+import Resolver
+
+extension ENExposureDetectionSummary {
+    var totalRiskScore: UInt {
+        let configurationService: ConfigurationService = Resolver.resolve()
+        var attenuationLevelValues = Set(configurationService.currentConfig.exposureConfiguration.attenuationLevelValues)
+
+        guard let attenuationLevels = attenuationLevelValues.unique() else {
+            let message = "The `exposure_configuration` object in the configuration.json is misconfigured"
+            assertionFailure(message)
+            LoggingService.error(message, context: .exposure)
+            return UInt.min
+        }
+
+        // 5 minutes in seconds
+        let detectionInterval: UInt = 300
```

Comment:
You can also write 
```suggestion
        let detectionInterval: UInt = 5 * 60
```
The compiler will create a constant for you.